docs/modules/ROOT/pages/architecture.adoc

```diff
@@ -1,7 +1,96 @@
+         |    | Lieutenant Operator +----+                                      |   | cBLU               |            |
+         |    | cBLU                |    |                                      |   \---------+----------/            |
+         |    \---------------------/    |  +--------+                          |             |                       |
+         |                               |  |        |                          |             |                       |
+         |                               +->+ Vault  +--------------------------------+-----  | Deployment            |
+         |                                  | {s}    |                          |     |       | Reconciliation        |
+         |  Get cluster, tenant and         +--------+                          |     |       |                       |
+         |  Git repo information                                                |     |       |                       |
+         |                                                                      |     |       v                       |
+         |  /------------------------\  Push compiled     +------------------+  |   +-+-------+----------+            |
+         |  |                        |  catalog           | Cluster catalog  |  |   |                    |            |
+         +--+ Commodore              +------------------->+ Git repo         +--------Argo-CD            |            |
+            | cBLU                   |                    |                  |  |   |                    |            |
+            \-----+------+--------+--/                    +------------------+  |   +--------------------+            |
+                  ^      ^        ^                                             |                                     |
+                  |      |        |                                             |                                     |
+          +-------+      +-----+  +--------------------+                        |                                     |
+          |                    |                       |                        |                                     |
++---------------------------------------------+        |                        |                                     |
+|         |                    |              |        |                        |                                     |
+| +-------+-------+ +----------+----------+   |  +-----+------+                 |                                     |
+| |               | |                     |   |  |            |                 |                                     |
+| | Common        | | Tenant and cluster  |   |  | Commodore  |                 |                                     |
+| | configuration | | configuration       |   |  | Components |                 |                                     |
```

Comment:
make all 3 git repos (common, tentant, catalog) the same color and/or include "Git repo" in all three boxes?